yt/yt/tests/integration/queues/test_queue_agent.py

```diff
@@ -3612,18 +3617,24 @@ def test_wrong_originating_queue(self):
 
     @authors("achulkov2", "nadya73")
     @pytest.mark.parametrize("use_upper_bound_for_table_names", [False, True])
-    def test_table_name_formatting(self, use_upper_bound_for_table_names):
+    @pytest.mark.parametrize("use_cron", [False, True])
+    def test_table_name_formatting(self, use_upper_bound_for_table_names, use_cron):
+        if use_cron and getattr(self, "USE_OLD_QUEUE_EXPORTER_IMPL"):
+            pytest.skip()
+
         export_dir = "//tmp/export"
         export_period_seconds = 3
 
         _, queue_id = self._create_queue("//tmp/q")
         self._create_export_destination(export_dir, queue_id)
+
         set("//tmp/q/@static_export_config", {
             "default": {
                 "export_directory": export_dir,
-                "export_period": export_period_seconds * 1000,
                 "output_table_name_pattern": "%ISO-period-is-%PERIOD-fmt-%Y.%m.%d.%H.%M.%S",
                 "use_upper_bound_for_table_names": use_upper_bound_for_table_names,
+                "export_cron_schedule": "0/3 * * * * *" if use_cron else None,
```

Comment:
Well, now `export_period_seconds` is unused :) 
Let's at least format it into the expression, like `f"0/{export_period_seconds} * * * * *`.

Also, you can make a helper function like `get_export_schedule(use_cron, export_period_seconds)` and then do something like this
```
{
    "export_directory": export_dir,
    ...
    **get_export_schedule(use_cron, export_period_seconds),
}
```

Python!